src/python/PSetTweaks/WMTweak.py

```diff
@@ -518,3 +520,90 @@ def makeOutputTweak(outMod, job):
     #      output module based on the settings in the section
 
     return result
+
+
+def shouldResizeResources():
```

Comment:
As commmented before (in the main PR), I'd suggest merging this code into resizeResources function, 90% of it is again in that func.